src/frontend/test_runner/tests/testdata/agg.yaml

```diff
@@ -148,10 +148,11 @@
           BatchScan { table: t, columns: [v1, v2] }
   stream_plan: |
     StreamMaterialize { columns: [agg#0(hidden), cnt, sum], pk_columns: [] }
-      StreamSimpleAgg { aggs: [count, count($0), sum($0)] }
+      StreamAppendOnlySimpleAgg { aggs: [sum($0), sum($1), sum($2)] }
```

Comment:
Why we always call the GlobalAgg as `AppendOnly`?

Reply:
Previously that was because I assumed local agg's to always be append_only. Have updated it.

See discussion: https://github.com/singularity-data/risingwave/pull/3320#discussion_r906628619.